### PR TITLE
HDDS-1836. Change the default value of ratis leader election min timeout to a lower value

### PR DESCRIPTION
**Pull request by bshashikant**

### CODE REVIEW

**Comment by hadoop-yetus** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 38 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | dupname | 0 | No case conflicting files found. |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| -1 | test4tests | 0 | The patch doesn't appear to include any new or modified tests.  Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch. |
||| _ trunk Compile Tests _ |
| +1 | mvninstall | 531 | trunk passed |
| +1 | compile | 254 | trunk passed |
| +1 | checkstyle | 68 | trunk passed |
| +1 | mvnsite | 0 | trunk passed |
| +1 | shadedclient | 821 | branch has no errors when building and testing our client artifacts. |
| +1 | javadoc | 159 | trunk passed |
| 0 | spotbugs | 319 | Used deprecated FindBugs config; considering switching to SpotBugs. |
| +1 | findbugs | 511 | trunk passed |
||| _ Patch Compile Tests _ |
| +1 | mvninstall | 481 | the patch passed |
| +1 | compile | 285 | the patch passed |
| +1 | javac | 285 | the patch passed |
| +1 | checkstyle | 70 | the patch passed |
| +1 | mvnsite | 0 | the patch passed |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | xml | 1 | The patch has no ill-formed XML file. |
| +1 | shadedclient | 639 | patch has no errors when building and testing our client artifacts. |
| +1 | javadoc | 163 | the patch passed |
| +1 | findbugs | 513 | the patch passed |
||| _ Other Tests _ |
| +1 | unit | 311 | hadoop-hdds in the patch passed. |
| -1 | unit | 2081 | hadoop-ozone in the patch failed. |
| +1 | asflicense | 54 | The patch does not generate ASF License warnings. |
| | | 7135 | |


| Reason | Tests |
|-------:|:------|
| Failed junit tests | hadoop.ozone.client.rpc.TestCommitWatcher |
|   | hadoop.ozone.client.rpc.TestCloseContainerHandlingByClient |
|   | hadoop.ozone.client.rpc.TestBlockOutputStreamWithFailures |
|   | hadoop.ozone.client.rpc.TestOzoneAtRestEncryption |
|   | hadoop.ozone.client.rpc.TestOzoneRpcClient |
|   | hadoop.ozone.client.rpc.TestFailureHandlingByClient |
|   | hadoop.hdds.scm.pipeline.TestRatisPipelineProvider |
|   | hadoop.ozone.container.server.TestSecureContainerServer |
|   | hadoop.ozone.client.rpc.TestSecureOzoneRpcClient |
|   | hadoop.ozone.client.rpc.TestOzoneClientRetriesOnException |
|   | hadoop.ozone.container.ozoneimpl.TestSecureOzoneContainer |
|   | hadoop.ozone.client.rpc.TestOzoneRpcClientWithRatis |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=18.09.8 Server=18.09.8 base: https://builds.apache.org/job/hadoop-multibranch/job/PR-1133/1/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/1133 |
| Optional Tests | dupname asflicense compile javac javadoc mvninstall mvnsite unit shadedclient findbugs checkstyle xml |
| uname | Linux 66a2ab8ea4d9 4.4.0-138-generic #164-Ubuntu SMP Tue Oct 2 17:16:02 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | personality/hadoop.sh |
| git revision | trunk / 7f1b76c |
| Default Java | 1.8.0_212 |
| unit | https://builds.apache.org/job/hadoop-multibranch/job/PR-1133/1/artifact/out/patch-unit-hadoop-ozone.txt |
|  Test Results | https://builds.apache.org/job/hadoop-multibranch/job/PR-1133/1/testReport/ |
| Max. process+thread count | 5119 (vs. ulimit of 5500) |
| modules | C: hadoop-hdds/common U: hadoop-hdds/common |
| Console output | https://builds.apache.org/job/hadoop-multibranch/job/PR-1133/1/console |
| versions | git=2.7.4 maven=3.3.9 findbugs=3.1.0-RC1 |
| Powered by | Apache Yetus 0.10.0 http://yetus.apache.org |


This message was automatically generated.

File: hadoop-hdds/common/src/main/resources/ozone-default.xml [hunk 1]
```diff
@@ -270,7 +270,7 @@
   </property>
   <property>
     <name>dfs.ratis.leader.election.minimum.timeout.duration</name>
-    <value>5s</value>
+    <value>2s</value>
     <tag>OZONE, RATIS, MANAGEMENT</tag>
     <description>The minimum timeout duration for ratis leader election.
```
Comment (xiaoyuyao) [inline hadoop-hdds/common/src/main/resources/ozone-default.xml]: Can you change the description to 2s as well?

**Comment by hadoop-yetus** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 38 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | dupname | 0 | No case conflicting files found. |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| -1 | test4tests | 0 | The patch doesn't appear to include any new or modified tests.  Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch. |
||| _ trunk Compile Tests _ |
| +1 | mvninstall | 588 | trunk passed |
| +1 | compile | 359 | trunk passed |
| +1 | checkstyle | 66 | trunk passed |
| +1 | mvnsite | 0 | trunk passed |
| +1 | shadedclient | 790 | branch has no errors when building and testing our client artifacts. |
| +1 | javadoc | 148 | trunk passed |
| 0 | spotbugs | 434 | Used deprecated FindBugs config; considering switching to SpotBugs. |
| +1 | findbugs | 637 | trunk passed |
||| _ Patch Compile Tests _ |
| +1 | mvninstall | 545 | the patch passed |
| +1 | compile | 353 | the patch passed |
| +1 | javac | 353 | the patch passed |
| +1 | checkstyle | 62 | the patch passed |
| +1 | mvnsite | 0 | the patch passed |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | xml | 1 | The patch has no ill-formed XML file. |
| +1 | shadedclient | 611 | patch has no errors when building and testing our client artifacts. |
| +1 | javadoc | 146 | the patch passed |
| +1 | findbugs | 649 | the patch passed |
||| _ Other Tests _ |
| -1 | unit | 303 | hadoop-hdds in the patch failed. |
| -1 | unit | 1613 | hadoop-ozone in the patch failed. |
| +1 | asflicense | 34 | The patch does not generate ASF License warnings. |
| | | 7083 | |


| Reason | Tests |
|-------:|:------|
| Failed junit tests | hadoop.ozone.client.rpc.TestOzoneRpcClient |
|   | hadoop.ozone.om.TestScmSafeMode |
|   | hadoop.ozone.client.rpc.TestOzoneAtRestEncryption |
|   | hadoop.ozone.client.rpc.TestSecureOzoneRpcClient |
|   | hadoop.ozone.client.rpc.TestBlockOutputStreamWithFailures |
|   | hadoop.ozone.client.rpc.TestCommitWatcher |
|   | hadoop.ozone.client.rpc.TestMultiBlockWritesWithDnFailures |
|   | hadoop.ozone.client.rpc.TestOzoneRpcClientWithRatis |
|   | hadoop.ozone.scm.pipeline.TestSCMPipelineMetrics |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.1 Server=19.03.1 base: https://builds.apache.org/job/hadoop-multibranch/job/PR-1133/2/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/1133 |
| Optional Tests | dupname asflicense compile javac javadoc mvninstall mvnsite unit shadedclient findbugs checkstyle xml |
| uname | Linux 0b498bf586aa 4.4.0-139-generic #165-Ubuntu SMP Wed Oct 24 10:58:50 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | personality/hadoop.sh |
| git revision | trunk / aebac6d |
| Default Java | 1.8.0_212 |
| unit | https://builds.apache.org/job/hadoop-multibranch/job/PR-1133/2/artifact/out/patch-unit-hadoop-hdds.txt |
| unit | https://builds.apache.org/job/hadoop-multibranch/job/PR-1133/2/artifact/out/patch-unit-hadoop-ozone.txt |
|  Test Results | https://builds.apache.org/job/hadoop-multibranch/job/PR-1133/2/testReport/ |
| Max. process+thread count | 5281 (vs. ulimit of 5500) |
| modules | C: hadoop-hdds/common U: hadoop-hdds/common |
| Console output | https://builds.apache.org/job/hadoop-multibranch/job/PR-1133/2/console |
| versions | git=2.7.4 maven=3.3.9 findbugs=3.1.0-RC1 |
| Powered by | Apache Yetus 0.10.0 http://yetus.apache.org |


This message was automatically generated.

**Comment by hadoop-yetus** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 84 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | dupname | 0 | No case conflicting files found. |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| -1 | test4tests | 0 | The patch doesn't appear to include any new or modified tests.  Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch. |
||| _ trunk Compile Tests _ |
| +1 | mvninstall | 638 | trunk passed |
| +1 | compile | 377 | trunk passed |
| +1 | checkstyle | 79 | trunk passed |
| +1 | mvnsite | 0 | trunk passed |
| +1 | shadedclient | 953 | branch has no errors when building and testing our client artifacts. |
| +1 | javadoc | 188 | trunk passed |
| 0 | spotbugs | 508 | Used deprecated FindBugs config; considering switching to SpotBugs. |
| +1 | findbugs | 730 | trunk passed |
||| _ Patch Compile Tests _ |
| +1 | mvninstall | 606 | the patch passed |
| +1 | compile | 426 | the patch passed |
| +1 | javac | 426 | the patch passed |
| +1 | checkstyle | 90 | the patch passed |
| +1 | mvnsite | 0 | the patch passed |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | xml | 2 | The patch has no ill-formed XML file. |
| +1 | shadedclient | 1065 | patch has no errors when building and testing our client artifacts. |
| +1 | javadoc | 187 | the patch passed |
| +1 | findbugs | 726 | the patch passed |
||| _ Other Tests _ |
| +1 | unit | 372 | hadoop-hdds in the patch passed. |
| -1 | unit | 2091 | hadoop-ozone in the patch failed. |
| +1 | asflicense | 42 | The patch does not generate ASF License warnings. |
| | | 8824 | |


| Reason | Tests |
|-------:|:------|
| Failed junit tests | hadoop.ozone.client.rpc.TestSecureOzoneRpcClient |
|   | hadoop.ozone.client.rpc.TestOzoneRpcClient |
|   | hadoop.ozone.client.rpc.TestBlockOutputStreamWithFailures |
|   | hadoop.ozone.client.rpc.TestOzoneAtRestEncryption |
|   | hadoop.ozone.om.TestScmSafeMode |
|   | hadoop.ozone.om.TestOzoneManagerHA |
|   | hadoop.ozone.client.rpc.TestOzoneRpcClientWithRatis |
|   | hadoop.hdds.scm.pipeline.TestRatisPipelineCreateAndDestory |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.0 Server=19.03.0 base: https://builds.apache.org/job/hadoop-multibranch/job/PR-1133/3/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/1133 |
| Optional Tests | dupname asflicense compile javac javadoc mvninstall mvnsite unit shadedclient findbugs checkstyle xml |
| uname | Linux a3a07e134eb0 4.15.0-52-generic #56-Ubuntu SMP Tue Jun 4 22:49:08 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | personality/hadoop.sh |
| git revision | trunk / d086d05 |
| Default Java | 1.8.0_212 |
| unit | https://builds.apache.org/job/hadoop-multibranch/job/PR-1133/3/artifact/out/patch-unit-hadoop-ozone.txt |
|  Test Results | https://builds.apache.org/job/hadoop-multibranch/job/PR-1133/3/testReport/ |
| Max. process+thread count | 4279 (vs. ulimit of 5500) |
| modules | C: hadoop-hdds/common U: hadoop-hdds/common |
| Console output | https://builds.apache.org/job/hadoop-multibranch/job/PR-1133/3/console |
| versions | git=2.7.4 maven=3.3.9 findbugs=3.1.0-RC1 |
| Powered by | Apache Yetus 0.10.0 http://yetus.apache.org |


This message was automatically generated.

**Comment by hadoop-yetus** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 62 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | dupname | 0 | No case conflicting files found. |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| -1 | test4tests | 0 | The patch doesn't appear to include any new or modified tests.  Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch. |
||| _ trunk Compile Tests _ |
| -1 | mvninstall | 351 | hadoop-ozone in trunk failed. |
| +1 | compile | 337 | trunk passed |
| +1 | checkstyle | 62 | trunk passed |
| +1 | mvnsite | 0 | trunk passed |
| +1 | shadedclient | 799 | branch has no errors when building and testing our client artifacts. |
| +1 | javadoc | 148 | trunk passed |
| 0 | spotbugs | 428 | Used deprecated FindBugs config; considering switching to SpotBugs. |
| +1 | findbugs | 618 | trunk passed |
||| _ Patch Compile Tests _ |
| +1 | mvninstall | 546 | the patch passed |
| +1 | compile | 379 | the patch passed |
| +1 | javac | 379 | the patch passed |
| +1 | checkstyle | 66 | the patch passed |
| +1 | mvnsite | 0 | the patch passed |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | xml | 2 | The patch has no ill-formed XML file. |
| +1 | shadedclient | 601 | patch has no errors when building and testing our client artifacts. |
| +1 | javadoc | 151 | the patch passed |
| +1 | findbugs | 613 | the patch passed |
||| _ Other Tests _ |
| +1 | unit | 360 | hadoop-hdds in the patch passed. |
| -1 | unit | 2477 | hadoop-ozone in the patch failed. |
| +1 | asflicense | 68 | The patch does not generate ASF License warnings. |
| | | 7909 | |


| Reason | Tests |
|-------:|:------|
| Failed junit tests | hadoop.ozone.TestSecureOzoneCluster |
|   | hadoop.ozone.client.rpc.TestSecureOzoneRpcClient |
|   | hadoop.ozone.client.rpc.TestMultiBlockWritesWithDnFailures |
|   | hadoop.ozone.client.rpc.TestOzoneAtRestEncryption |
|   | hadoop.hdds.scm.pipeline.TestRatisPipelineCreateAndDestory |
|   | hadoop.ozone.client.rpc.TestBlockOutputStreamWithFailures |
|   | hadoop.ozone.client.rpc.TestOzoneRpcClient |
|   | hadoop.ozone.om.TestScmSafeMode |
|   | hadoop.ozone.om.TestKeyManagerImpl |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.1 Server=19.03.1 base: https://builds.apache.org/job/hadoop-multibranch/job/PR-1133/4/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/1133 |
| Optional Tests | dupname asflicense compile javac javadoc mvninstall mvnsite unit shadedclient findbugs checkstyle xml |
| uname | Linux b762a0a4074d 4.4.0-138-generic #164-Ubuntu SMP Tue Oct 2 17:16:02 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | personality/hadoop.sh |
| git revision | trunk / 70b4617 |
| Default Java | 1.8.0_222 |
| mvninstall | https://builds.apache.org/job/hadoop-multibranch/job/PR-1133/4/artifact/out/branch-mvninstall-hadoop-ozone.txt |
| unit | https://builds.apache.org/job/hadoop-multibranch/job/PR-1133/4/artifact/out/patch-unit-hadoop-ozone.txt |
|  Test Results | https://builds.apache.org/job/hadoop-multibranch/job/PR-1133/4/testReport/ |
| Max. process+thread count | 5072 (vs. ulimit of 5500) |
| modules | C: hadoop-hdds/common U: hadoop-hdds/common |
| Console output | https://builds.apache.org/job/hadoop-multibranch/job/PR-1133/4/console |
| versions | git=2.7.4 maven=3.3.9 findbugs=3.1.0-RC1 |
| Powered by | Apache Yetus 0.10.0 http://yetus.apache.org |


This message was automatically generated.

**Comment by hadoop-yetus** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 65 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | dupname | 0 | No case conflicting files found. |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| -1 | test4tests | 0 | The patch doesn't appear to include any new or modified tests.  Please justify why no new tests are needed for this patch. Also please list what manual steps were performed to verify this patch. |
||| _ trunk Compile Tests _ |
| +1 | mvninstall | 576 | trunk passed |
| +1 | compile | 354 | trunk passed |
| +1 | checkstyle | 64 | trunk passed |
| +1 | mvnsite | 0 | trunk passed |
| +1 | shadedclient | 792 | branch has no errors when building and testing our client artifacts. |
| +1 | javadoc | 149 | trunk passed |
| 0 | spotbugs | 422 | Used deprecated FindBugs config; considering switching to SpotBugs. |
| +1 | findbugs | 616 | trunk passed |
||| _ Patch Compile Tests _ |
| +1 | mvninstall | 537 | the patch passed |
| +1 | compile | 356 | the patch passed |
| +1 | javac | 356 | the patch passed |
| +1 | checkstyle | 65 | the patch passed |
| +1 | mvnsite | 0 | the patch passed |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | xml | 1 | The patch has no ill-formed XML file. |
| +1 | shadedclient | 619 | patch has no errors when building and testing our client artifacts. |
| +1 | javadoc | 147 | the patch passed |
| +1 | findbugs | 644 | the patch passed |
||| _ Other Tests _ |
| +1 | unit | 368 | hadoop-hdds in the patch passed. |
| -1 | unit | 2495 | hadoop-ozone in the patch failed. |
| +1 | asflicense | 39 | The patch does not generate ASF License warnings. |
| | | 8020 | |


| Reason | Tests |
|-------:|:------|
| Failed junit tests | hadoop.ozone.client.rpc.TestHybridPipelineOnDatanode |
|   | hadoop.hdds.scm.pipeline.TestNodeFailure |
|   | hadoop.ozone.web.client.TestKeys |
|   | hadoop.ozone.client.rpc.TestBlockOutputStream |
|   | hadoop.ozone.client.rpc.TestMultiBlockWritesWithDnFailures |
|   | hadoop.hdds.scm.pipeline.TestRatisPipelineCreateAndDestory |
|   | hadoop.hdds.scm.pipeline.TestRatisPipelineProvider |
|   | hadoop.hdds.scm.pipeline.TestNode2PipelineMap |
|   | hadoop.ozone.client.rpc.TestBCSID |
|   | hadoop.ozone.client.rpc.TestContainerStateMachine |
|   | hadoop.hdds.scm.safemode.TestSCMSafeModeWithPipelineRules |
|   | hadoop.ozone.client.rpc.TestCloseContainerHandlingByClient |
|   | hadoop.ozone.client.rpc.TestReadRetries |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.1 Server=19.03.1 base: https://builds.apache.org/job/hadoop-multibranch/job/PR-1133/5/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/1133 |
| Optional Tests | dupname asflicense compile javac javadoc mvninstall mvnsite unit shadedclient findbugs checkstyle xml |
| uname | Linux cf50d74aba0e 4.4.0-138-generic #164-Ubuntu SMP Tue Oct 2 17:16:02 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | personality/hadoop.sh |
| git revision | trunk / 397a563 |
| Default Java | 1.8.0_222 |
| unit | https://builds.apache.org/job/hadoop-multibranch/job/PR-1133/5/artifact/out/patch-unit-hadoop-ozone.txt |
|  Test Results | https://builds.apache.org/job/hadoop-multibranch/job/PR-1133/5/testReport/ |
| Max. process+thread count | 3745 (vs. ulimit of 5500) |
| modules | C: hadoop-hdds/common U: hadoop-hdds/common |
| Console output | https://builds.apache.org/job/hadoop-multibranch/job/PR-1133/5/console |
| versions | git=2.7.4 maven=3.3.9 findbugs=3.1.0-RC1 |
| Powered by | Apache Yetus 0.10.0 http://yetus.apache.org |


This message was automatically generated.

**Pull request closed without merging**